### PR TITLE
Add description to EIP

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
This pull request add the description attribute to `exoscale_ipadress` resource

### CODE REVIEW

**Comment by pierre-emmanuelJ** [comment]
=== RUN   TestAccResourceIPAddress
--- PASS: TestAccResourceIPAddress (10.31s)
PASS
ok  	github.com/terraform-providers/terraform-provider-exoscale/exoscale	10.330s

**Comment by pierre-emmanuelJ** [comment]
I'll update `egoscale` dep when it will be released

File: exoscale/resource_exoscale_ipaddress_test.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ func TestAccResourceIPAddress(t *testing.T) {
 						"healthcheck_timeout":      ValidateString(fmt.Sprint(testIPHealthcheckTimeout1)),
 						"healthcheck_strikes_ok":   ValidateString(fmt.Sprint(testIPHealthcheckStrikesOk1)),
 						"healthcheck_strikes_fail": ValidateString(fmt.Sprint(testIPHealthcheckStrikesFail1)),
+						"description":              ValidateString("IPAdress 1"),
```
Comment (falzm) [inline exoscale/resource_exoscale_ipaddress_test.go]: Please don't hardcode test values but rather follow the existing pattern of declaring constants (e.g. `testIPDescription1 = "IP address 1"`) and reference them throughout the tests: https://github.com/pierre-emmanuelJ/terraform-provider-exoscale/blob/ch6590/eip-add-description/exoscale/resource_exoscale_ipaddress_test.go#L13-L28

**Review by falzm — CHANGES_REQUESTED** [review]

File: exoscale/resource_exoscale_ipaddress_test.go (hunk 1 above)
Comment (pierre-emmanuelJ) [reply exoscale/resource_exoscale_ipaddress_test.go]: thx

**Comment by pierre-emmanuelJ** [comment]
Vendor is updated on latest egoscale release

**Review by falzm — APPROVED** [review]
LGTM, thanks @pierre-emmanuelJ 👍

**Review by mcorbin — APPROVED** [review]

**Comment by falzm** [comment]
@pierre-emmanuelJ acceptance actually fails:

```
------- Stdout: -------
=== RUN   TestAccResourceIPAddress
--- FAIL: TestAccResourceIPAddress (13.07s)
    testing.go:569: Step 0 error: Check failed: Check 3/3 error: invalid value for attribute "description":
        string "" doesn't match expected value "IPAdress 1"
        
FAIL
```

**Review by falzm — CHANGES_REQUESTED** [review]
Acceptance test fails:

```
------- Stdout: -------
=== RUN   TestAccResourceIPAddress
--- FAIL: TestAccResourceIPAddress (13.07s)
    testing.go:569: Step 0 error: Check failed: Check 3/3 error: invalid value for attribute "description":
        string "" doesn't match expected value "IPAdress 1"
        
FAIL
```

**Comment by pierre-emmanuelJ** [comment]
Only in preprod for the moment 😕

**Comment by pierre-emmanuelJ** [comment]
Was not in prod when you have tested, only in preprod. 

```
=== RUN   TestAccResourceIPAddress
--- PASS: TestAccResourceIPAddress (9.95s)
PASS
ok  	github.com/terraform-providers/terraform-provider-exoscale/exoscale	9.977s
```

**Comment by falzm** [comment]
> Was not in prod when you have tested, only in preprod.

OK thanks.

**Review by falzm — APPROVED** [review]
LGTM, thanks @pierre-emmanuelJ

**Review by pyr — APPROVED** [review]

**Pull request merged**